src/plugins/gui/RewardsCardPlugin.tsx

```diff
@@ -19,9 +19,10 @@ import { initializeProviders } from './util/initializeProviders'
 
 const SUPPORT_URL = 'https://edge.app/visa-card-how-to'
 
-export interface RewardsCardItem {
```

Comment:
with one day left in 3.11.0 staging testing. I feel uncomfortable with this refactor (although I do want the refactor in 3.12). I'd say this task gets punted to 3.12.0 or undo the refactor